src/test/javascript/spec/service/orion/orion-assessment.service.spec.ts

```diff
@@ -111,48 +107,53 @@ describe('OrionAssessmentService', () => {
 
         testConversion(mockReader as any);
```

Comment:
I assume this is a cheat here? Because in the helper function we expect a FileReader

Reply:
Yeah, this is the same as with the undefined. In this case I did not define a full reader but only created an object mocking the relevant functions, so we need to cheat the typecheck again

Reply:
COuld you add the same comment as for the other occurrence here?

Reply:
Then please document this decision in a short comment above the expect statement. If that should be the "// ignore RepositoryExportOptions", this is not clear enough. Explicitly state that you refer to the expect.anything(). Basically like you did here, thanks.

Reply:
Could you add the same comment as for the other occurrence here?

Reply:
I now changed the parameter type so the any isnt needed anymore